libcontainer/cgroups/fs/utils.go

```diff
@@ -55,12 +55,17 @@ func getCgroupParamKeyValue(t string) (string, uint64, error) {
 
 // Gets a single uint64 value from the specified cgroup file.
 func getCgroupParamUint(cgroupPath, cgroupFile string) (uint64, error) {
-	contents, err := ioutil.ReadFile(filepath.Join(cgroupPath, cgroupFile))
+	fileName := filepath.Join(cgroupPath, cgroupFile)
+	contents, err := ioutil.ReadFile(fileName)
 	if err != nil {
 		return 0, err
 	}
 
-	return parseUint(strings.TrimSpace(string(contents)), 10, 64)
+	res, err := parseUint(strings.TrimSpace(string(contents)), 10, 64)
+	if err != nil {
+		return res, fmt.Errorf("unable to parse %q as a uint from Cgroup file %q", string(contents), fileName)
```

Comment:
@duglin Just curious. Is there any situation that you actually got an error here so you want more details? Because this is not supposed to fail here.

Anyway, more detailed error message is always better.